criu/protobuf.c

```diff
@@ -48,14 +47,15 @@ static char *image_name(struct cr_img *img)
 
 int do_pb_read_one(struct cr_img *img, void **pobj, int type, bool eof)
 {
+	char imgp[PATH_MAX];
```

Comment:
could you

Reply:
This was my first choice, but then it makes some of the pr_err to wrap line. Should I still rename?

Reply:
This was my first choice, but then it makes some of the pr_err to wrap line. Should I still rename?

Reply:
what do you think about this https://github.com/avagin/criu/commit/797f27605bbd7f9ebf0bd15a5c4e23305fac80dc ?

Reply:
sold